### PR TITLE
Add global method to get records that satisfy all filters (#64)

### PR DESCRIPTION
**Pull request by betalb**
Implemented method allFiltered as discussed in #64

### CODE REVIEW

File: src/crossfilter.js [hunk 1]
```diff
@@ -1328,6 +1329,21 @@ function crossfilter() {
     return data;
   }
 
+  // Returns row data with all dimension filters applied
+  function allFiltered() {
+    var array = [],
+        i = 0;
+
+      while (i < n) {
```
Comment (esjewett) [inline src/crossfilter.js]: Generally the Crossfilter code base uses for-loops. Can you switch over to `for(i=0;i<n;i++)` to align?

**Comment by esjewett** [comment]
Looks good, but I would prefer to use a for-loop to align to the general style in the rest of the code base.

@tannerlinsley @gordonwoodhull - You 2 OK with this?

**Comment by gordonwoodhull** [comment]
LGTM. I think it will be helpful to have this, and it makes the interface somewhat more symmetrical.

**Comment by esjewett** [comment]
Thanks, and sorry for the delay on this. Merging and it will be included in the next alpha release (which will probably be a beta release, since things seem pretty stable).

**Pull request merged**